README.md

```diff
@@ -120,6 +120,23 @@ Use Makefile generator which is default one.
     cmake .
     make
 
+## Testing
+
+This package includes a collection of automated tests.
+Running them is not part of building nor installation but optional. 
+And they can only run under UNIX-like system.
```

Comment:
Does `ctest` not work on Windows?

Reply:
My mistake. `ctest` can't no run directly under WIndows, but can run with option `-C Debug|Release`. I had modifed REAME.md.